dapps/evancore.vue.libs/src/components/forms/form/form.ts

```diff
@@ -211,20 +211,40 @@ export default class EvanFormComponent extends mixins(EvanComponent) {
   getTranslation(control: EvanFormControl, type: string) {
     // if manual error text was specified, translate it and return it directly
     if (type === 'error') {
-      if (typeof control.error !== 'boolean') {
-        return this.$t(control.error);
-      } else if (!control.error) {
-        return control.error;
+      if (typeof control[type] !== 'boolean') {
```

Comment:
why change it and lose TS support, when type is always checked before if it's === 'error' ?